<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="toolBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>71</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="invertCircle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="invertCircleIcon">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="drawLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="invertCircleInSquare">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cutImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="invertCircleIcon">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>641</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>инструменты</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>инвертировать в окружности</string>
     </property>
     <addaction name="invertUsingCircleMenu"/>
     <addaction name="invertUsingSquareMenu"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="drawLineMenu"/>
    <addaction name="cutMenu"/>
    <addaction name="backMenu"/>
   </widget>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>дополнительно</string>
    </property>
    <addaction name="info"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="invertUsingCircleMenu">
   <property name="text">
    <string>через окружность</string>
   </property>
  </action>
  <action name="invertUsingSquareMenu">
   <property name="text">
    <string>через квадрат</string>
   </property>
  </action>
  <action name="drawLineMenu">
   <property name="text">
    <string>нарисовать линию</string>
   </property>
  </action>
  <action name="cutMenu">
   <property name="text">
    <string>вырезать часть изображения</string>
   </property>
  </action>
  <action name="info">
   <property name="text">
    <string>справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="backMenu">
   <property name="text">
    <string>отменить изменения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
